cli/src/torii/tests.rs

```diff
@@ -56,16 +70,17 @@ async fn create_torii() -> (Torii<World>, KeyPair) {
         .is_none());
     wsv.world.domains.insert(domain_id, domain);
     let queue = Arc::new(Queue::from_configuration(&config.queue, Arc::clone(&wsv)));
-    let network = IrohaNetwork::new(
-        Broker::new(),
-        config.torii.p2p_addr.clone(),
-        config.public_key.clone(),
-        config.network.actor_channel_capacity,
-    )
-    .await
-    .expect("Failed to create network")
-    .start()
-    .await;
+
+    let network = loop {
+        // this loop is not infinite, because we call unwrap on unique_port::get_unique_free_port(),
+        // which leads to panic if all available ports are already in use
+        match try_init_network(&mut config).await {
```

Comment:
I'd be interested to know how many tries on average it takes for tests when run with nextest.